### PR TITLE
52349_Create product

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52349)

## WHAT (optional)
- Create product

## HOW
- use gem carrierwave to upload image

## Evidence (Screenshot or Video)
Screenshot from 2022-08-18 10-41-34
Screenshot from 2022-08-18 10-24-23

### CODE REVIEW

**Review by diemntn-0165 — APPROVED** [review]

**Comment by fonqhoanq** [comment]
ready

File: app/controllers/admin/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.new product_params
+    if @product.save
+      flash[:success] = t ".success"
+      redirect_to new_admin_product_url
```
Comment (ngocvt-0484) [inline app/controllers/admin/products_controller.rb]:
```suggestion
      redirect_to new_admin_product_path
```

File: app/helpers/admin/categories_helper.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Admin::CategoriesHelper
+  def category_collect
```
Comment (ngocvt-0484) [inline app/helpers/admin/categories_helper.rb]: remove

File: app/controllers/admin/products_controller.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    else
+      flash.now[:error] = t ".error"
+      render :new
+    end
+  end
+
+  private
+  def product_params
+    params.require(:product)
+          .permit(:name, :description, :price,
```
Comment (ngocvt-0484) [inline app/controllers/admin/products_controller.rb]: params đặt biến constant trong model

**Comment by ngocvt-0484** [comment]
cho folder public/upload vào gitignore

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**